crates/query-engine/translation/Cargo.toml

```diff
@@ -16,3 +16,4 @@ serde_json = "1.0.108"
 [dev-dependencies]
 insta = { version = "1.34.0", features = ["json"] }
 sqlformat = "0.2.2"
+ndc-sdk = { git = "https://github.com/hasura/ndc-hub.git", rev = "fb03873" }
```

Comment:
It is so we can reference the type `MutationRequest` defined in the `models` of `ndc-spec` for parsing purposes. We didn't have to do that for queries because type inference worked in that particular case.

Reply:
Hm, interestingly the automatic merging failed because the ndc-sdk version specified in `[dependencies]` had drifted.

So I tried simply removing the `[dev-dependencies]` mention of it, and it built and passed CI and eventually merged automatically.

I think `[dev-dependencies]` are only for development tools not necessarily related to compilation (such as `insta`, as already appears in that section).